examples/proc_combined_ODNP.py

```diff
@@ -0,0 +1,332 @@
+import os, time, h5py
+import pylab as plt
+from numpy import empty
+from sympy import exp as s_exp
+from sympy import symbols, Symbol
+from matplotlib.ticker import FuncFormatter
+import matplotlib.transforms as transforms
+from pyspecdata import *
+from pyspecProcScripts import *
+from pyspecProcScripts import lookup_table
```

Comment:
the point of this is just to check the data -- so while this might form the basis of a larger code on procScripts, you don't want all the bells and whistles.